Description: Use system default window manager instead of openbox
Author: Laurent Bigonville <bigon@debian.org>
Forwarded: not-needed

--- a/sandbox/sandbox
+++ b/sandbox/sandbox
@@ -268,7 +268,7 @@ class Sandbox:
             copyfile(f, "/tmp", self.__tmpdir)
             copyfile(f, "/var/tmp", self.__tmpdir)
 
-    def __setup_sandboxrc(self, wm="/usr/bin/openbox"):
+    def __setup_sandboxrc(self, wm="/usr/bin/x-window-manager"):
         execfile = self.__homedir + "/.sandboxrc"
         fd = open(execfile, "w+")
         if self.__options.session:
@@ -362,7 +362,7 @@ sandbox [-h] [-l level ] [-[X|M] [-H hom
 
         parser.add_option("-W", "--windowmanager", dest="wm",
                           type="string",
-                          default="/usr/bin/openbox",
+                          default="/usr/bin/x-window-manager",
                           help=_("alternate window manager"))
 
         parser.add_option("-l", "--level", dest="level",
--- a/sandbox/sandbox.8
+++ b/sandbox/sandbox.8
@@ -77,7 +77,7 @@ Specifies the windowsize when creating a
 \fB\-W\fR \fB\-\-windowmanager\fR
 Select alternative window manager to run within 
 .B sandbox \-X.
-Default to /usr/bin/openbox.
+Default to /usr/bin/x-window-manager.
 .TP
 \fB\-X\fR 
 Create an X based Sandbox for gui apps, temporary files for
